sklearn/tests/test_pipeline.py

```diff
@@ -983,6 +983,71 @@ def test_set_feature_union_step_drop():
     assert not record
 
 
+def test_set_feature_union_passthrough():
+    """Check the behaviour of setting a transformer to `"passthrough"`."""
+    mult2 = Mult(2)
+    mult2.get_feature_names = lambda: ["x2"]
+    mult3 = Mult(3)
+    mult3.get_feature_names = lambda: ["x3"]
```

Comment:
Those `get_feature_names` methods are not use in the test, no? If so, let's remove them.

Reply:
Or alternatively, expand the test to use them.

Reply:
I removed the get_feature_names